src/screens/OrganizationEvents/CustomRecurrenceModal.tsx

```
@@ -0,0 +1,596 @@
+        ? new Date(endDate.getTime() + 7 * 24 * 60 * 60 * 1000)
+        : new Date(Date.now() + 30 * 24 * 60 * 60 * 1000);
+
+      setRecurrenceRuleState((prev) => ({
+        ...prev,
+        never: false,
+        count: undefined,
+        endDate: defaultRecurrenceEndDate,
+      }));
+    } else if (selectedOption === endsAfter) {
+      const totalCount =
+        typeof localCount === 'string' ? parseInt(localCount) : localCount;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)